content/en/platform/corda/5.1/deploying-operating/observability/metrics/task-manager.md

```diff
@@ -0,0 +1,55 @@
+| `corda_taskmanager_executor_active_threads`               | Gauge   | `name`                                                    | The approximate number of threads that are actively executing tasks.                   |
+| `corda_taskmanager_executor_completed_tasks_total`        | Counter | `name`                                                    | The approximate number of tasks that have completed execution.                         |
+| `corda_taskmanager_executor_idle_seconds_count`           | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_idle_seconds_max`             | Gauge   | `name`                                                    |                                                                                        |
+| `corda_taskmanager_executor_idle_seconds_sum`             | Counter | `name`                                                    | The total sum of all observed values for the base metric.                           |
+| `corda_taskmanager_executor_pool_core_threads`            | Gauge   | `name`                                                    | The core number of threads for the pool.                                               |
+| `corda_taskmanager_executor_pool_max_threads`             | Gauge   | `name`                                                    | The maximum allowed number of threads in the pool.                                     |
+| `corda_taskmanager_executor_pool_size_threads`            | Gauge   | `name`                                                    | The current number of threads in the pool.                                             |
+| `corda_taskmanager_executor_queue_remaining_tasks`        | Gauge   | `name`                                                    | The number of additional elements that this queue can ideally accept without blocking. |
+| `corda_taskmanager_executor_queued_tasks`                 | Gauge   | `name`                                                    | The approximate number of tasks that are queued for execution.                         |
+| `corda_taskmanager_executor_scheduled_repetitively_total` | Counter | `name`                                                    |                                                                                        |
+| `corda_taskmanager_executor_seconds_count`                | Counter | `name`                                                    | The number of events that have been observed for the base metric.                   |
+| `corda_taskmanager_executor_seconds_max`                  | Gauge   | `name`                                                    |                                                                                        |
```

Comment:
@lankydan do we have a description for this one?

Reply:
"The max observed value for the base metric" I guess.